lib/matplotlib/axis.py

```diff
@@ -1203,14 +1203,16 @@ def _update_ticks(self):
 
         return ticks_to_draw
 
-    def _get_ticklabel_bboxes(self, ticks, renderer):
+    def _get_ticklabel_bboxes(self, ticks, renderer=None):
         """Return lists of bboxes for ticks' label1's and label2's."""
+        if renderer is None:
```

Comment:
I don't think you need to normalize here; just let the labels do it?

Reply:
That depends not he strategy we want to employ - it could wait to normalize until the end, but this does save a bunch of calls to `_get_renderer`.

Reply:
Fair.